limited_area/region_spec.py

```diff
@@ -109,12 +109,41 @@ def gen_spec(self, fileName, *args, **kwargs):
                                                     self.in_point[0],
                                                     self.in_point[1])
 
-
             # Convert to meters, then divide by radius to get radius upon sphere w/ r = 1
             self.radius = (self.radius * 1000) / EARTH_RADIUS
             self.points = self.circle(self.in_point[0], self.in_point[1], self.radius)
 
             return self.name, self.in_point, [self.points.flatten()]
+        elif self.type == 'channel':
+            if self._DEBUG_ > 0:
+                print("DEBUG: Using the channel method for region generation")
+
+            if self.ulat == self.llat:
+                print("ERROR: Upper and lower lattiude for channel specification")
+                print("ERROR: can not be equal")
```

Comment:
It may be a matter of style, but I think `cannot` (as a single word) is preferable to `can not`.

Reply:
Cannot is definitely one word. Thanks for the checks on spelling.